docs/content/events.md

```diff
@@ -13,9 +13,9 @@ Events are for broadcasting information between worker instances about a transie
 
 ### Sending events
 
-A worker instance can send an event using a `EventSender<T>` ECS component (where `T` is the SpatialOS component that the event is defined in).
+A worker instance can send an event using a `ComponentName.EventSenders.EventName` ECS component, where `ComponentName` is the name of the component that the event is defined in and `EventName` is the name of the event in schema.
 
```

Comment:
"..in, and.." 
In this circumstance the "in" followed by the "and" trips you up when you are reading it. A comma will help.